app/src/main/java/es/wolfi/utils/CredentialLabelSort.java

```diff
@@ -0,0 +1,29 @@
+package es.wolfi.utils;
+
+import java.util.Comparator;
+
+import es.wolfi.passman.API.Credential;
+
+/**
+ * Created by wolfi on 12/11/17.
+ */
+
+public class CredentialLabelSort implements Comparator<Credential> {
+
+    private final int method;
+
+    public CredentialLabelSort(int method) {
+        this.method = method;
+    }
+
+    @Override
+    public int compare(Credential left, Credential right) {
+        if (method == 1) {
```

Comment:
What does one and two mean? What's the default behaviour? I think this class could use a bit more documentation overall.

Reply:
changed it to constants with a14e0e817e204c647a8bf6bd77a4ed4a4ee0cee6